templates/firefox-ios.xliff

```diff
@@ -1754,12 +1770,12 @@
         <note>Show Firefox Browser Privacy Policy page from the Privacy section in the settings. See https://www.mozilla.org/privacy/firefox/</note>
       </trans-unit>
       <trans-unit id="ProtectionStatusSheet.InsecureConnection" xml:space="preserve">
-        <source>Insecure Connection</source>
+        <source>Connection is not secure</source>
         <note>value for label to indicate user is on an unencrypted website</note>
```

Comment:
String ID should change here because this is a new string.

Reply:
Instead of updating the string ID, I've made new strings.